jerry-core/ecma/base/ecma-gc.c

```diff
@@ -852,11 +858,11 @@ ecma_gc_mark (ecma_object_t *object_p) /**< object to mark from */
             }
           }
 
-          jmem_cpointer_t proto_cp = object_p->u2.prototype_cp;
+          jmem_cpointer_t array_proto_cp = object_p->u2.prototype_cp;
 
-          if (proto_cp != JMEM_CP_NULL)
+          if (array_proto_cp != JMEM_CP_NULL)
```

Comment:
I think we dont need that local variable. `array_proto_cp` is eqivalent to `proto_cp ` above.

Reply:
True. PR updated